profile/README.md

```diff
@@ -0,0 +1,11 @@
+# Welcome to Pulsar
+
+Pulsar, or sometimes referred to as Pulsar-Edit is a new image of the beloved 'Hackable Text Editor' [Atom](https://github.com/atom/atom).
+
+After the announcement of [Atom's sunset](https://github.blog/2022-06-08-sunsetting-atom/) the community came together to keep Atom alive, over on a longstanding fork [Atom-Community](https://github.com/atom-community/atom).
+
+Although due to the difference in goals for the editor long term a new version was born Pulsar.
+
+Pulsar aims to not only reach feature parity with origin Atom, but to bring Pulsar into the 21st century. With updates to the underlying dependencies and modern features supported.
+
+With many new features intended to come once things are stable Pulsar will be a true Community-Based Hackable Text Editor.
```

Comment:
With many new features on the roadmap, once Pulsar is stable, it will be a true, Community-Based, Hackable, Text Editor.